uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -973,32 +982,32 @@ sub createNewCandID {
 
 =pod
 
-B<getPSC( C<$patientName>, C<$dbhr> )>
+B<getPSC( C<$identifier>, C<$dbhr> )>
 
-Look for the site alias in whatever field (usually patient name or patient_id) is provided 
+Look for the site alias in whatever field (usually patient name or patient_id) is provided
 and return the MRI alias and CenterID
 
 Returns: two element array: first is UNKN or the MRI alias of the PSC, second is the centerID or 0
 
 =cut
 
 sub getPSC {
-    my ($patientName, $dbhr) = @_;
+    my ($identifier, $dbhr) = @_;
     my $query = "SELECT CenterID, Alias, MRI_alias FROM psc WHERE mri_alias<>''";
     my $sth = $${dbhr}->prepare($query);
     $sth->execute;
 
     while(my $row = $sth->fetchrow_hashref) {
         return ($row->{'MRI_alias'}, $row->{'CenterID'})
-	    if ($patientName =~ /$row->{'Alias'}/i) || ($patientName =~ /$row->{'MRI_alias'}/i);
+	    if ($identifier =~ /$row->{'Alias'}/i) || ($identifier =~ /$row->{'MRI_alias'}/i);
     }
 
-    ###########################################################################	
+    ###########################################################################
     ###Get the centerID using the PSCID########################################
     ###########################################################################
 
     #extract the PSCID from $patientName
-    $subjectIDsref = getSubjectIDs($patientName,null,$dbhr);
```

Comment:
I don't think we need it in MRI.pm, but I haven't tested it yet. Will do so shortly.